dist/docker/redhat/Dockerfile

```diff
@@ -4,6 +4,9 @@ MAINTAINER Avi Kivity <avi@cloudius-systems.com>
 
 ENV container docker
 
+# default argument for scylla repo url, will be used incase no external argument provided
+ARG SCYLLA_REPO_URL=http://downloads.scylladb.com/rpm/centos/scylla-3.1.repo
```

Comment:
The value is still incorrect. It needs to be `http://downloads.scylladb.com/rpm/unstable/centos/master/latest/scylla.repo` like the one below which you are replacing.

Reply:
Sorry,, fixed